Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule_test.py

```diff
@@ -665,10 +668,28 @@ def test_parse_autonomous_system_indicator(self, taxii_2_client):
             "id": "autonomous-system--f720c34b-98ae-597f-ade5-27dc241e8c74",
             "number": 15139,
             "name": "Slime Industries",
-            "rir": "ARIN"
+            "rir": "ARIN",
+            "extensions": {"extension-definition--1234": {"CustomFields": {"tags": ["test"], "description": "test"}}}
         }
 
         xsoar_expected_response = [
+            {
+                'value': 15139,
+                'score': Common.DBotScore.NONE,
+                'rawJSON': autonomous_system_obj,
+                'type': 'ASN',
+                'fields': {
+                    'description': 'test',
+                    'firstseenbysource': '',
+                    'modified': '',
+                    'name': 'Slime Industries',
+                    'stixid': 'autonomous-system--f720c34b-98ae-597f-ade5-27dc241e8c74',
+                    'tags': ["test"],
+                    'trafficlightprotocol': 'GREEN'
+                }
+            }
+        ]
+        xsoar_expected_response_with_update_custom_fields = [
```

Comment:
Shouldn't this be `xsoar_expected_response` and the object in line 675 `xsoar_expected_response_with_update_custom_fields`?